backend/config.yaml

```
@@ -0,0 +1,152 @@
+# Kruzna Karta Hrvatska - Backend Configuration
+# This file contains default values that can be overridden by environment variables
+
+# Database Configuration
+database:
+  host: "${DB_HOST:localhost}"
+  port: "${DB_PORT:5432}"
+  name: "${DB_NAME:kruzna_karta_hrvatska}"
+  user: "${DB_USER:postgres}"
+  password: "${DB_PASSWORD:}"
+  url: "${DATABASE_URL:postgresql://${DB_USER:postgres}:${DB_PASSWORD:}@${DB_HOST:localhost}:${DB_PORT:5432}/${DB_NAME:kruzna_karta_hrvatska}}"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)